frontend/src/config/globals.jsx

```diff
@@ -1,5 +1,4 @@
 export const Constants = {
-  api: process.env.REACT_APP_API_URL,
```

Comment:
We should keep the old changes here. On localhost we will read the JSON from a different location and we should change easily the endpoint.

Reply:
Added it back.